functests/utils/nodes/nodes.go

```diff
@@ -32,6 +33,21 @@ func GetByRole(c client.Client, role string) ([]corev1.Node, error) {
 	return nodes.Items, nil
 }
 
+// GetNonCNFNode retuns node with no worker-rt label
+func GetNonCNFNode(c client.Client, role string) ([]corev1.Node, error) {
+	nonRTWorkerNode := []corev1.Node{}
+
+	workerNodes, err := GetByRole(testclient.Client, role)
+	for _, node := range workerNodes {
+		if _, ok := node.Labels[fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerRT)]; ok {
+			continue
+		}
+		nonRTWorkerNode = append(nonRTWorkerNode, node)
+		break
```

Comment:
i think it's better to remove **break**
Without it we will have full list of noncnf nodes, it might be more useful then returning only one node

Reply:
Ack. Done.